src/AlarmClock-Spec/AlarmClockPresenter.class.st

```diff
@@ -0,0 +1,58 @@
+"
+A presenter for an instance of AlarmClock.
+"
+Class {
+	#name : #AlarmClockPresenter,
+	#superclass : #SpPresenter,
+	#instVars : [
+		'model',
+		'elapsedTimeLabel',
+		'finishedLabel'
+	],
+	#category : #'AlarmClock-Spec'
+}
+
+{ #category : #specs }
+AlarmClockPresenter class >> defaultSpec [
+	^ SpBoxLayout newVertical
+		add: #elapsedTimeLabel;
+		add: #finishedLabel;
```

Comment:
Should set a height

Reply:
Done

Reply:
Thanks, the default height is too large and ugly.